app/src/main/java/com/sopt/kakaobank/presentation/home/HomeFragment.kt

```diff
@@ -1,11 +1,78 @@
 package com.sopt.kakaobank.presentation.home
 
+import android.graphics.Rect
+import android.os.Bundle
+import android.telephony.PhoneNumberUtils.formatNumber
+import android.util.TypedValue
+import android.view.View
+import android.widget.AdapterView
+import android.widget.AdapterView.OnItemClickListener
+import androidx.databinding.DataBindingUtil.setContentView
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import com.sopt.kakaobank.R
 import com.sopt.kakaobank.core.base.BindingFragment
 import com.sopt.kakaobank.databinding.FragmentHomeBinding
+import com.sopt.kakaobank.databinding.ItemBankBook1Binding
+import com.sopt.kakaobank.presentation.history.HistoryFragment
+import java.text.NumberFormat
+import java.util.Locale
 
 class HomeFragment : BindingFragment<FragmentHomeBinding>(R.layout.fragment_home) {
+
+    private val viewModel: HomeViewModel by viewModels()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.homeViewModel = viewModel
+        binding.lifecycleOwner = viewLifecycleOwner
+        initRecyclerView()
+    }
+
+    private fun initRecyclerView() {
+        val formattedHomeItems = viewModel.homeItems.map { item ->
+            when (item) {
+                is HomeItem.BankBookItem3 -> {
+                    val formattedWithdraw = formatNumber(item.withdraw)
+                    item.copy(withdraw = formattedWithdraw)
+                }
+                else -> item
+            }
+        }.toMutableList()
+
+        val adapter = HomeItemAdapter(formattedHomeItems, this)
+        binding.homeRecyclerView.layoutManager = LinearLayoutManager(context)
+        binding.homeRecyclerView.adapter = adapter
+
+        // recyclerview item 간 간격 조정
+        val spaceInPixels = TypedValue.applyDimension(
+            TypedValue.COMPLEX_UNIT_DIP, 9f, resources.displayMetrics
+        ).toInt()
+
+        binding.homeRecyclerView.addItemDecoration(object : RecyclerView.ItemDecoration() {
+            override fun getItemOffsets(
+                outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State
+            ) {
+                outRect.bottom = spaceInPixels
+            }
+        })
```

Comment:
core 패키지에 ItemDiffCallback 파일을 생성해두었어요. 제 PR 보고 참고해서 적용해보면 좋을 것 같아요! 그렇게 하면 ItemDecorator 파일을 따로 생성하여 아래 처럼 코드를 사용할 수 있어요.
```suggestion
        binding.homeRecyclerView.addItemDecoration(HomeRecyclerViewItemDecorator(requireContext()))
```

Reply:
오 저도 참고해서 적용해봐야겠네요! 감사합니다!